tests/ert_tests/ensemble_evaluator/test_sync_ws_duplexer.py

```diff
@@ -68,7 +67,8 @@ async def handler(websocket, path):
 
     duplexer.stop()
 
-    assert closed
+    closed.wait(10)
+    assert closed.is_set()
```

Comment:
Why not just `assert closed.wait(10)`  ?

Reply:
Hey, hadn't read that part of the documentation. Thanks!